src/Features/CSharp/Portable/Completion/KeywordRecommenders/WhereKeywordRecommender.cs

```diff
@@ -85,7 +83,7 @@ private static bool IsTypeParameterConstraintContext(CSharpSyntaxContext context
             if (baseList?.Parent is TypeDeclarationSyntax typeDecl)
             {
                 if (typeDecl.TypeParameterList != null &&
-                    typeDecl.BaseList.Types.Any(t => token == t.GetLastToken(includeSkipped: true)))
+                    typeDecl.BaseList!.Types.Any(t => token == t.GetLastToken(includeSkipped: true)))
```

Comment:
Does this have the same semantics but avoids the suppression?

```suggestion
                    baseList.Types.Any(t => token == t.GetLastToken(includeSkipped: true)))
```